samples/trophies/trophies/build.bat

```diff
@@ -0,0 +1,80 @@
+SETLOCAL EnableDelayedExpansion
+
+Rem Package information
+set PKG_TITLE="OpenOrbis Trophy Sample"
+set PKG_VERSION="1.00"
+set PKG_TITLE_ID="BREW00080"
```

Comment:
Title ID and content ID need updating, BREW00080 conflicts with audio wav sample. Recommend BREW00094. Makefile also needs to be written for samples for linux compatibility.